File: 0125-valid-palindrome/0125-valid-palindrome.java
class Solution {
    public boolean isPalindrome(String s) {
        String g="";
        for(String i:s.split(""))
        {
            if(i.matches("[A-Za-z0-9]+"))
            {
                g=g+i.toLowerCase();
            }
            
        }
        String l="";
        for(String i:g.split(""))
        {
            l=i+l;
        }
        
        if(s.equals(" "))
        {
            return true;
        }
        else if(g.equals(l))
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}

File: 0125-valid-palindrome/0125-valid-palindrome.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        t=""
        for i in s:
            if i.isalnum():
                t=t+i.lower()
        
        j=t[::-1]
        if s==" ":
            return True
        elif(t==j):
            return True
        else:
            return False
        

        